Server/db/MongoDB.js

```diff
@@ -414,9 +414,13 @@ const createCheck = async (checkData) => {
  * @throws {Error}
  */
 
-const getChecks = async (monitorId) => {
+const getChecks = async (monitorId, limitedTo) => {
```

Comment:
@ajhollid Is this okay?

Reply:
Sure, but I think you can name that var `limit`

Reply:
Done.
how about other parts?

Reply:
Have you updated the route to include the query param?  I don't think I saw that here

Reply:
Frontend will also need to be updated to make the appropriate request, and docs need to be updated too